fastlane/lib/fastlane/runner.rb

```diff
@@ -109,7 +109,9 @@ def find_alias(action_name)
 
     # This is being called from `method_missing` from the Fastfile
     # It's also used when an action is called from another action
-    def trigger_action_by_name(method_sym, custom_dir, *arguments)
+    # @param from_action Indicates if this action is being trigged by another action.
+    #                    If so, it won't show up in summary.
+    def trigger_action_by_name(method_sym, custom_dir, from_action, *arguments)
```

Comment:
In the past we've had trouble with adding parameters to existing functions breaking people's setups, so could you give this a default value please? e.g. `def trigger_action_by_name(method_sym, custom_dir, from_action: false, *arguments)`